File: POO/Python/uberPool.py
#/usr/bin/env python3.8
from carrito import Car
class UberPool(Car): 
    brand = str
    model = str
    def __init__(self, license, driver, brand, model):
        super().__init__(license, driver)
        self.brand = brand
        self.model = model
    def printDataCar(self):
        print(self.license)

File: POO/PHP/uberPool.php
<?php
require_once("car.php");
    class UberPool extends Car{
        public $model;
        public $brand;
        public function __construct($license, $driver, $brand, $model){
            parent::__construct($license, $driver);
            $this->model = $model;
            $this->brand = $brand;
        }
    }
?>